README.md

```diff
@@ -100,3 +100,30 @@ To fetch, for example, the root squashfs, you can simply use `scp`:
 ```
 $ scp ubuntu@<INSTANCE>:images/livecd.ubuntu-cpc.squashfs .
 ```
+
+## Customising the Built Images
+
+In order to customise the contents of the built images, you can provide
+a second argument to `generate_build_config.py`.  This should be the
+path to a shell script which will be run within the chroot after all
+other image building is complete.
+
+For example, if you wanted RabbitMQ server to be installed in all the
+images that are produced, you could write a shell script that looked
+something like this out to `script.sh`:
+
+```
+#!/bin/sh
```

Comment:
I would suggest adding "-x" to match the other output the user will see.

Also an echo won't hurt as an example of a good practice to say we're here.  The build output will have no delimiter saying that it's executing this hook so debugging will be harder for the end user without some breadcrumbs.

`echo "======================== Running customization script ==========================="`